db/sql/table02_users.sql

```diff
@@ -1,5 +1,6 @@
 CREATE TABLE users (
-    user_id text      NOT NULL UNIQUE PRIMARY KEY CHECK (40 <= LENGTH(user_id) AND LENGTH(user_id) <= 55),
-    created timestamp WITH time zone NOT NULL DEFAULT now()
+    id              serial PRIMARY KEY,
```

Comment:
I'd suggest we change it so that each table has an `id` field which is it's `id`, then other tables having things like `user_id` or `pursuance_id` are known to be FK's. What do you think?